README.md

```diff
@@ -508,6 +505,78 @@ cd $CMSSW_BASE/src
 scram b -j 8
 ```
 
+</details>
+
+### Instructions for 124X
+
+```
+## IMPORTANT ##
+# Make sure your architecture is slc7_amd64_gcc10
+# (you can set the architecture with: export SCRAM_ARCH=slc7_amd64_gcc700 )
+
+cmsrel CMSSW_12_4_14_patch2
+cd CMSSW_12_4_14_patch2/src
+cmsenv
+
+git cms-init
+
+# EGamma Tools - Electrons Id, Iso and Energy Scale
+# https://twiki.cern.ch/twiki/bin/view/CMS/EgammaUL2016To2018
+
+git cms-addpkg RecoEgamma/EgammaTools  ### essentially just checkout the package from CMSSW
+git clone https://github.com/cms-egamma/EgammaPostRecoTools.git
+mv EgammaPostRecoTools/python/EgammaPostRecoTools.py RecoEgamma/EgammaTools/python/.
```

Comment:
Suggestion: replace two lines above with  
```
wget https://raw.githubusercontent.com/cms-sw/cmssw/e988bd296b885193909d05157b51e6f8ba0ba5e9/RecoEgamma/EgammaTools/python/EgammaPostRecoTools.py
mv EgammaPostRecoTools.py RecoEgamma/EgammaTools/python/
```
and remove the comment below

Reply:
Will be changed in the next commit